### PR TITLE
Sort community tournaments in the `Tournaments` index page by game modes

### LINKED ISSUE #5740: Extra sorting and headings for the `Community` section on the `Tournaments` page
[The tournaments page](https://osu.ppy.sh/wiki/en/Tournaments) seems to be getting a bit messy to navigate as well as slowly becoming more unsorted so I want to propose adding in some extra headings to more clearly differentiate between gamemodes as well as sorting all the tournaments by alphabetical order for the community section. 

Contents would look like:
```
Official World Cups
  osu! World Cup
  osu!taiko World Cup
  osu!catch World Cup
  osu!mania World Cup
Community
  osu!
    (Alphabetised list of osu! tournaments)
  osu!taiko
    (Alphabetised list of osu!taiko tournaments)
  osu!catch
    (Alphabetised list of osu!catch tournaments)
  osu!mania
    (Alphabetised list of osu!mania tournaments)
  Multimode
    (Alphabetised list of multimode tournaments)
```

Tournaments already seem to be separated by mode but there is no border between the two, some headings would make it easier to distinguish the changeover. While sorting by alphabetical order mostly applies to osu! tournaments, I think it would help to have this written down somewhere.

There could be a case made for splitting the tournament wiki section into more distinct mode sections, I don't know how much easier it would be to actually find info.

### PR DESCRIPTION
**Pull request by TPGPL**
Resolves #5740

### CODE REVIEW

File: wiki/Tournaments/en.md [hunk 1]
```diff
@@ -94,176 +100,213 @@ Unofficial tournaments/competitions hosted by the communities.
...
 
-### ![](/wiki/shared/mode/osu.png) Coffee Hispanic
+#### osu! European Tournament
 
 | Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
-| [Coffee Hispanic Tournament](CH/CHT_1) | 2020-01-17 | 2020-02-17 | United States | Noveno Gaston | hallo y caspi |
-| [Coffee Hispanic Showdown](CH/CHS_1) | 2020-08-06 | 2020-09-13 | yotambien | carlos | catJAM |
+| osu! European Tournament | 2018-04-07 | 2018-05-27 | a | Dumb and Dumber | MinolaureMVP |
+| [osu! European Tournament 2](o!ET/2) | 2019-02-10 | 2019-03-30 | Sruba (Top), Makeshift Winners (Mid), not even european (Low) | Almost Allies (Top), singletap is braindead (Mid), Yes (Yes) (Low) | nh9mju0 (Top), Phoenix Emperors (Mid), Polish Musketeers (Low) |
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:174]:
```suggestion

#### osu! Indonesia Tournament

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [osu! Indonesia Tournament #10 (2020)](OIT/2020) | 2020-02-03 | 2020-04-12 | ![][flag_ID] [Skydiver](https://osu.ppy.sh/users/4750008) | ![][flag_ID] [Fuma](https://osu.ppy.sh/users/1501956) | ![][flag_ID] [Rexeez](https://osu.ppy.sh/users/1987591) |
|[osu! Indonesia Tournament #11 (2021)](OIT/2021) | 2021-01-04 | 2021-03-28 | ![][flag_ID] [rho](https://osu.ppy.sh/users/1629553) | ![][flag_ID] [Skydiver](https://osu.ppy.sh/users/4750008) | ![][flag_ID] [Crezz](https://osu.ppy.sh/users/7108275) |

```

File: wiki/Tournaments/en.md [hunk 2]
```diff
@@ -352,177 +389,148 @@ Unofficial tournaments/competitions hosted by the communities.
...
+| [osu!mania Asian's Champions 2019](o!mAC/2019) | 2019-09-28 | 2019-11-10 | ![][flag_KR] [\[ R o a d \]](https://osu.ppy.sh/users/6535926) | ![][flag_TW] [\[Crz\]FolAH1217](https://osu.ppy.sh/users/6232458) | ![][flag_MY] [xxxxxx2800](https://osu.ppy.sh/users/4084853) |
 
-### ![](/wiki/shared/mode/mania.png) osu!mania Asian's Champions
+#### osu!mania Flashlight 4K Tournament
 
 | Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
-| [osu!mania Asian's Champions 2019](o!mAC/2019) | 2019-09-28 | 2019-11-10 | ![][flag_KR] [\[ R o a d \]](https://osu.ppy.sh/users/6535926) | ![][flag_TW] [\[Crz\]FolAH1217](https://osu.ppy.sh/users/6232458) | ![][flag_MY] [xxxxxx2800](https://osu.ppy.sh/users/4084853) |
+| [osu!mania Flashlight 4K Tournament](FL4T/1) | 2021-02-06 | 2021-03-21 | ![][flag_BR] [andreymc](https://osu.ppy.sh/users/5691061) | ![][flag_GB] [TingMomentum](https://osu.ppy.sh/users/12489832) | ![][flag_PH] [- l e j -](https://osu.ppy.sh/users/6413944) |
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:451]:
```suggestion

#### osu!mania Indonesia Cup

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [osu!mania 4K Indonesia Cup 2019](OMIC/2019_4K) | 2019-05-26 | 2019-08-25 | ![][flag_ID] [Kevin-sama](https://osu.ppy.sh/users/6991664) | ![][flag_ID] [-\[ChoKy\]-](https://osu.ppy.sh/users/4691146) | ![][flag_ID] [ChocoCoki-](https://osu.ppy.sh/users/3565870) |

```

File: wiki/Tournaments/en.md [hunk 3]
```diff
@@ -352,177 +389,148 @@ Unofficial tournaments/competitions hosted by the communities.
...
-| [2v2 :thonking: Mania Tournament](TMT/2TMT) | 2018-11-10 | 2018-12-23 | La pancit bois | 3 argentos y medio | PolytETral |
+| [Mania Beginner's Showdown](MBS/1) | 2019-06-15 | 2019-08-04 | ![][flag_SE] [Davvyforte](https://osu.ppy.sh/users/10047413) (High division), ![][flag_TH] [osu\_player420](https://osu.ppy.sh/users/13018880) (Low division) | ![][flag_DE] [Maou](https://osu.ppy.sh/users/3867109) (High division), ![][flag_PH] [Wakeupat5](https://osu.ppy.sh/users/13311553) (Low division) | ![][flag_US] [Loopy542](https://osu.ppy.sh/users/5468461) (High division), ![][flag_VE] [\_Vanslight\_](https://osu.ppy.sh/users/6031119) (Low division) |
+| [Mania Beginner's Showdown 2](MBS/2) | 2020-05-09 | 2020-06-05 | ![][flag_PH] [Lazereed](https://osu.ppy.sh/users/12894120) (High division), ![][flag_MY] [Anamneses](https://osu.ppy.sh/users/16299729) (Low division) | ![][flag_FR] [Paturages](https://osu.ppy.sh/users/1375479) (High division), ![][flag_SE] [Mestro](https://osu.ppy.sh/users/4798263) (Low division) | ![][flag_SG] [loli\_seale](https://osu.ppy.sh/users/5069913) (High division), ![][flag_MX] [Ammy](https://osu.ppy.sh/users/4183406) (Low division) |
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:433]:
```suggestion

#### Mengxin Cup

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [Novice Mania Cup 3rd](MXB/3) | 2019-04-05 | 2019-06-16 | ![][flag_CN] [A M D](https://osu.ppy.sh/users/5321112) | ![][flag_CN] [shizehao](https://osu.ppy.sh/users/4928674) | ![][flag_CN] [Yutto007](https://osu.ppy.sh/users/10269172) |

#### Mistrzostwa Polski (Polish Championship)

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [Mistrzostwa Polski w 4K 2019](MP/2019_4K) | 2019-05-13 | 2019-08-04 | ![][flag_PL] [Tidek](https://osu.ppy.sh/users/743282) | ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946) | ![][flag_PL] [_underjoy](https://osu.ppy.sh/users/2235750) |
| [Mistrzostwa Polski w 7K 2019](MP/2019_7K) | 2019-10-09 | 2019-12-22 | ![][flag_PL] [- gero -](https://osu.ppy.sh/users/5068537) | ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783) | ![][flag_PL] [-Karen-chan-](https://osu.ppy.sh/users/1333495) |
| [Mistrzostwa Polski w 7K 2020](MP/2020_7K) | 2020-10-12 | 2020-12-20 | ![][flag_PL] [paprotkaaaaa](https://osu.ppy.sh/users/6034749) | ![][flag_PL] [- gero -](https://osu.ppy.sh/users/5068537) | ![][flag_PL] [_underjoy](https://osu.ppy.sh/users/2235750) |

```

File: wiki/Tournaments/en.md [hunk 4]
```diff
@@ -94,176 +100,213 @@ Unofficial tournaments/competitions hosted by the communities.
...
+| [UK osu!standard tournament!](OUKT/1) | 2013-11-11 | 2014-1-11 | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [R a h a r u](https://osu.ppy.sh/users/785193) | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) |
+| [UK osu!standard tournament 2](OUKT/2) | 2014-03-15 | 2014-04-20 | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [SoraRoxas01](https://osu.ppy.sh/users/1986262) |
+| [osu! UK Tournament 3](OUKT/3) | 2014-06-28 | 2014-08-10 | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [jesus1412](https://osu.ppy.sh/users/230116) | ![][flag_GB] [iLikeMudkipz](https://osu.ppy.sh/users/552515) |
+| [osu! UK Tournament 4](OUKT/4) | 2015-01-10 | 2015-08-21 | ![][flag_GB] [bahamete](https://osu.ppy.sh/users/960620) | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | N/A |
+| [osu! UK Tournament 5](OUKT/5) | 2015-09-12 | 2016-04-23 | ![][flag_GB] [bahamete](https://osu.ppy.sh/users/960620) | ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | ![][flag_GB] [Kakiage](https://osu.ppy.sh/users/3104031) |
+
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:222]:
```suggestion

#### osu! Z-Tournament

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [osu! Z-Tournament 2018](OZT/2018) | 2018-04-23 | 2018-07-15 | Floating Rubber | LatexIntolerance | goose is a furry |

#### PERAK osu! Tournament

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [PERAK osu! Tournament](POT/1) | 2021-01-17 | 2021-04-04 | ![][flag_ID] [ReeyaDono](https://osu.ppy.sh/users/10969698) | ![][flag_ID] [Lindaine](https://osu.ppy.sh/users/4572717) | ![][flag_ID] [Victim\_Crasher](https://osu.ppy.sh/users/2084869) |

```

File: wiki/Tournaments/en.md [hunk 5]
```diff
@@ -94,176 +100,213 @@ Unofficial tournaments/competitions hosted by the communities.
...
+| [Royal Danish osu!Cup 2020](RDOC/2020) | 2020-07-01 | 2020-09-27 | ![][flag_DK] [iamVill](https://osu.ppy.sh/users/6295380) | ![][flag_DK] [Pablohh](https://osu.ppy.sh/users/10092099) | ![][flag_DK] [Vandabe](https://osu.ppy.sh/users/7050754) |
+
+#### SS Tournament
 
-### ![](/wiki/shared/mode/osu.png) Ulat Bulu Cup
+| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| SS Tournament Summer 2017 | 2017-07-23 | 2017-09-24 | Content is hot | Yaong's kawaii sisters~ | Majiyabakune |
+| [SS Tournament 2018](SST/2018) | 2018-01-27 | 2018-03-18 | Content is Hot | Sleeping Accommodation | Floating up |
+
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:253]:
```suggestion

#### TAG2 Tournament

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [TAG2 Tournament 2019](T2T/2019) | 2019-02-15 | 2019-05-19 | Thank You | osu! schmosu! | yoo whats goodie eh |

```

File: wiki/Tournaments/en.md [hunk 6]
```diff
@@ -352,177 +389,148 @@ Unofficial tournaments/competitions hosted by the communities.
...
 
-### ![](/wiki/shared/mode/mania.png) osu!mania Flashlight 4K Tournament
+#### The Heptakey osu!mania Cup
 
 | Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
-| [osu!mania Flashlight 4K Tournament](FL4T/1) | 2021-02-06 | 2021-03-21 | ![][flag_BR] [andreymc](https://osu.ppy.sh/users/5691061) | ![][flag_GB] [TingMomentum](https://osu.ppy.sh/users/12489832) | ![][flag_PH] [- l e j -](https://osu.ppy.sh/users/6413944) |
-
-### ![](/wiki/shared/mode/osu.png) ![](/wiki/shared/mode/catch.png) ![](/wiki/shared/mode/mania.png) osu!Philippines Summer Tournament
+| [The Heptakey osu!mania Cup #1](THOM_CUP/1) | 2018-04-16 | 2018-07-15 | ![][flag_ES] [aitor98](https://osu.ppy.sh/users/3154852) | ![][flag_IT] [Yomiel](https://osu.ppy.sh/users/3461860) | ![][flag_CL] [Skalim](https://osu.ppy.sh/users/2225008) |
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:491]:
```suggestion

#### Touhou Project Mania Cup

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [Touhou Project Mania Cup 1st](TMC/1st) | 2020-09-14 | 2020-12-06 | MY 2.5 + TH 1.5 | yuyupa | SMOrcat |

```

File: wiki/Tournaments/en.md [hunk 7]
```diff
@@ -278,59 +321,53 @@ Unofficial tournaments/competitions hosted by the communities.
...
 
-### ![](/wiki/shared/mode/taiko.png) 3 Digit Taiko World Cup
+#### Taiko Wars
 
 | Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
 | :-- | :-- | :-- | :-- | :-- | :-- |
-| [3 Digit Taiko World Cup](3TWC/1) | 2021-01-09 | 2021-02-21 | ![][flag_CN] China | ![][flag_KR] South Korea | ![][flag_MY] Malaysia |
-
-### ![](/wiki/shared/mode/taiko.png) North & South American Taiko Tournament
+| [Taiko Wars](TW/1) | 2018-10-29 | 2018-12-02 | Land of Foda-se 2: Electric Boogaloo | cu | Iceland |
```
Comment (Bazzadwarf) [inline wiki/Tournaments/en.md:357]:
```suggestion

#### Vaf's Autumn Fiesta

| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
| :-- | :-- | :-- | :-- | :-- | :-- |
| [Vaf's Autumn Fiesta](VAF/1) | 2021-04-16 | 2021-07-11 | *Undecided* | *Undecided* | *Undecided* |

```

**Review by Bazzadwarf — COMMENTED** [review]
Might as well add in the tournaments that are currently missing now rather than later.

**Comment by TPGPL** [comment]
applied, nice work 👍

**Review by TicClick — APPROVED** [review]
looks ok

**Pull request merged**